.gitignore

```diff
@@ -0,0 +1,2 @@
+# .gitignore
+node_modules/
```

Comment:
- [ ] Kindly, make sure that you add your `test.md` file in your `,gitignore` file to ignore it in your remote repository.